docs/Community/Repositories.md

```diff
@@ -0,0 +1,124 @@
+      unless otherwise approved by the [CGC](governance/README.md).
+   4. Must follow other [contributing guidelines](Contributing.md)
+      unless otherwise approved by the [CGC](governance/README.md).
+   5. Must live under `github.com/openenclave/<project-name>`.
+   6. Code repositories must be approved by
+      [SIG-Architecture](sig-architecture/README.md).
+      Non-code repositories must be approved by the
+      [CGC](governance/README.md).
+
+### Temporary forks
+
+Sometimes the Open Enclave project will want changes to other external
+repositories.  Ideally, such changes would be contributed
+directly to the upstream project.  However, in some cases it may be
+necessary to use a temporary fork for some period of time, whether for
+development, or for use while the upstream submission is being reviewed
+and iterated on.  In such cases, a fork might be done under an individual's
+github organization or the openenenclave organization.  (A fork is not
+needed at all if a branch can be created in the related project.)
+
+If the Open Enclave SDK needs to directly depend on a fork of the related
+project, it is generally better for the fork to be done under the openenclave
+organization.  This makes it clear that the fork is a dependency for Open Enclave
+SDK and is not intended for general public use.
+
+#### Rules
+
+The repository must:
+
+   1. Adopt the Open Enclave [Code of Conduct](Governance.md#code-of-conduct)
+      statement in their repo.
+   2. Use the same license and copyright rules as the forked project.
+   3. Adopt the [DCO](Contributing.md#developer-certificate-of-origin)
+      bot automation for pull requests
+      unless otherwise approved by the [CGC](governance/README.md).
+   4. Follow the other [contributing guidelines](Contributing.md)
+      unless otherwise approved by the [CGC](governance/README.md).
+   5. Be created under `github.com/openenclave/<project-name>`.
+   6. Be approved by [SIG-Architecture](sig-architecture/README.md).
+
+Other than rule 2, all other rules are the same as for core repositories.
+
+## Creating Repositories
+
+Requests for creating repositories can be made by filing an
+[Open Enclave issue](https://github.com/openenclave/openenclave/issues)
+if one does not already exist.
+
+In addition to stating any deviations from the Open Enclave defaults
+discussed under Rules above, any request to create a new repository
+should also answer the following questions:
+
+1. How should issues with the new repo be tracked and triaged?
+   Should github issues be filed in the new repo, or in the main
+   openenclave repo?  Should they be discussed in the master openenclave
+   triage meeting, or in some separate process?
+2. Should one or more of the existing SIGs (e.g.,
+   [SIG-Architecture](sig-architecture/README.md),
+   [SIG-Testing](sig-testing/README.md)) oversee relevant aspects of the
+   new repo, or not?
+3. Who is reponsible for maintaining the new repo?  How is their
```

Comment:
I suggest the recording format for this 3rd item be standardized, so that the maintainer-list be more easily integrated with any relevant project-wide automation.

Reply:
The way the [oeedger8r](https://github.com/openenclave/oeedger8r-cpp/blob/master/OWNERS) project handles this regarding the "who" I believe is ideal and it ties into automation easily.

Reply:
Opening #3640 for standardization of the third.